### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by alexlaurinmath**
Adds checks and tests to ensure working software as soon as it lands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,99 @@
...
+- Boolean
+- String
+- Integer
+- Decimal
+- Quantity
+- Date
+- Time
+- DateTime
+
+FHIR Protos get implicitly to the above types according to this
```
Comment (VickSuresh) [inline README.md:91]:
```suggestion
FHIR Protos get implicitly converted to the above types according to this
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,99 @@
...
+[chart](http://hl7.org/fhir/R4/fhirpath.html#types), when used in some FHIRPath expressions.
+
+### Things to be aware of
+
+FHIRPath is not the most intuitive language, and there are some quirks. See [gotchas](gotchas.md).
```
Comment (VickSuresh) [inline README.md:96]: We're currently missing `gotchas.md`

**Review by VickSuresh — APPROVED** [review]
LGTM once comments are addressed

File: README.md (hunk 1 above)
Comment (alexlaurinmath) [reply README.md:91]: I'll close this branch and make the changes in the fork.

File: README.md (hunk 2 above)
Comment (alexlaurinmath) [reply README.md:96]: I'll close this branch and make the changes in the fork.

**Pull request closed without merging**